packages/exchange/src/adapters/base-url-adapter.ts

```diff
@@ -0,0 +1,8 @@
+export const baseUrlAdapter = (baseUrl: string, providerId: string) => {
+  switch (providerId) {
+    case 'encryptus':
```

Comment:
just for history: Encryptus has this limitation on their end where the path MUST end with `/` which is not part of ABNF for URIs definition.